### ISSUE TITLE
Web - Step 6 - Topic 2 - Preventing Recurrence - Complex Data Tables Needs IDs and HEADERS

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#79]
A complex data table's data cells are not associated with its headers programmatically. Example: The last TABLE needs to have IDs, and HEADERs added because it's a complex TABLE since the last six rows in the first column span the last cell in the second column.

Here is a set of my slides that explains how to do a complex TABLE with ID and HEADERS along with a helpful article - http://jfciii.com/presentations/wasp/accessibility.html

Step 6 - Topic 2 - Preventing Recurrence - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/steps/6/topic/2

**Comment by KyleLivingstonDOL** [comment]
ZTI, can we address the table data cells?

**Comment by jfc3-dol** [comment]
Thanks for adding the TH scope="col" for the two-column headings, but there is now TH scope="row" code on the first content items of each row for the first three TABLEs. The last TABLE needs to user IDs and HEADERS to get this to work properly because there are six rows that SPAN one at the bottom of the TABLE.

Having both allows the assistive technology user (screen reader, etc.) to know what column heading and row information that a particular cell is related too.

**Comment by KyleLivingstonDOL** [comment]
ZTI, can we please correct (linked to Issue 78)

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17

Here is an article on [how to make TABLEs accessible](
https://www.456bereastreet.com/archive/200410/bring_on_the_tables/) from a simple TABLE using scope="col" and scope="row" on the TH elements to complex TABLEs at the bottom of the article using Th with an ID and then a HEADERS with the proper ID information on the correct TD items. 

**NOTE**: You can have more than one ID in a HEADERS.

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17

The issues looks to be Step 8 Topic 3 now - https://dolcontentdev.opadev.dol.gov/ilab/complychain/steps/8/topic/3 But I will double check once the website is updated with the latest code.

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17

The last TABLE on the page has been rebuilt not to have a column span of six rows. So I'm going to close the issue.

While looking at the code, I noticed the TABLEs now have a tabindex="0" on all the TABLE cells, which means a keyboard user has to tab through all of them. So I'm going to create a new issue to fix that.
